exchanges/account/account_test.go

```diff
@@ -292,8 +296,20 @@ func TestBalanceInternalWait(t *testing.T) {
 func TestBalanceInternalLoad(t *testing.T) {
 	t.Parallel()
 	bi := &ProtectedBalance{}
-	bi.load(Balance{Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
+	err := bi.load(nil)
+	assert.ErrorIs(t, err, common.ErrNilPointer, "should error nil pointer correctly")
+
+	err = bi.load(&Balance{Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
+	assert.ErrorIs(t, err, errUpdatedAtIsZero, "should have not been loaded")
+
+	now := time.Now()
+	err = bi.load(&Balance{UpdatedAt: now, Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
+	assert.NoError(t, err, "should have been loaded")
+
 	bi.m.Lock()
+	if bi.updatedAt.IsZero() {
+		t.Fatal("unexpected value")
+	}
```

Comment:
🚧 Can we check equality please?
```suggestion
    	assert.Equal(t, now, bi.updatedAt, "updatedAt should contain the correct value")
```
Or alternatively:
```
    	assert.False(t, bi.updatedAt.IsZero(), "updatedAt should not be IsZero")
```
Or finally:
```
    	assert.NotEmpty(t, bi.updatedAt, "updatedAt should contain a value")
```